as-vending/main.go

```diff
@@ -70,40 +72,40 @@ func main() {
 
 	var err error
 
-	err = edgexSdk.AddRoute("/boardStatus", vendingState.BoardStatus, "POST")
-	errorAddRouteHandler(edgexSdk, err)
+	err = service.AddRoute("/boardStatus", vendingState.BoardStatus, "POST")
+	errorAddRouteHandler(lc, err)
 
-	err = edgexSdk.AddRoute("/resetDoorLock", vendingState.ResetDoorLock, "POST")
-	errorAddRouteHandler(edgexSdk, err)
+	err = service.AddRoute("/resetDoorLock", vendingState.ResetDoorLock, "POST")
+	errorAddRouteHandler(lc, err)
 
-	err = edgexSdk.AddRoute("/maintenanceMode", vendingState.GetMaintenanceMode, "GET", "OPTIONS")
-	errorAddRouteHandler(edgexSdk, err)
+	err = service.AddRoute("/maintenanceMode", vendingState.GetMaintenanceMode, "GET", "OPTIONS")
+	errorAddRouteHandler(lc, err)
 
 	// create the function pipeline to run when an event is read on the device channels
-	err = edgexSdk.SetFunctionsPipeline(
-		transforms.NewFilter(vendingState.Configuration.DeviceNames).FilterByDeviceName,
+	err = service.SetFunctionsPipeline(
+		transforms.NewFilterFor(vendingState.Configuration.DeviceNames).FilterByDeviceName,
 		vendingState.DeviceHelper,
 	)
 	if err != nil {
-		edgexSdk.LoggingClient.Error("SDK initialization failed: " + err.Error())
-		os.Exit(-1)
+		lc.Errorf("SDK initialization failed: %s", err.Error())
+		os.Exit(1)
 	}
 
 	// tell the SDK to "start" and begin listening for events to trigger the pipeline.
-	err = edgexSdk.MakeItRun()
+	err = service.MakeItRun()
 	if err != nil {
-		edgexSdk.LoggingClient.Error("MakeItRun returned error: ", err.Error())
-		os.Exit(-1)
+		lc.Errorf("MakeItRun returned error: %s", err.Error())
+		os.Exit(1)
 	}
 
 	// do any required cleanup here
 
 	os.Exit(0)
 }
 
-func errorAddRouteHandler(edgexSdk *appsdk.AppFunctionsSDK, err error) {
+func errorAddRouteHandler(lc logger.LoggingClient, err error) {
 	if err != nil {
-		edgexSdk.LoggingClient.Error("Error adding route: %v", err.Error())
-		os.Exit(-1)
+		lc.Errorf("Error adding route: %s", err.Error())
+		os.Exit(1)
```

Comment:
Move to Controller wrapper

Reply:
I will wrap in controller in next PR as enhancement